nss-day-cohort-39/tabloid-full-stack-bobs
# Bb subscribe

### To test:
- Make a query to the Subscription table
- Run the app and go to a post's details page
- Click the blue subscribe button at the bottom of the page. It should change to a red unsubscribe button
- Run your subscription query again. There should be a new subscription with the user and author id, with a start date matching the current date and an end date of "9999-01-01T00:00:00"
- Click the unsubscribe button. It should change back to the subscribe button
- Run your query again. The corresponding subscription's end date should now match the current date
- Repeat with different posts
